### PR TITLE
Use dotenv to configure environment

### PR DESCRIPTION
**Pull request by ciex**
This lets us configure environment options with a file `.env`. Defaults for this file are contained in `.env.default`, which is also loaded to set a default for all config options.

This PR also simplifies commands in `package.json` where environment variables were heavily used before. In particular, the build of the KatasterKI-app and region-specific options can now be configured through the .env-file.

The `CONFIG_ENV`-option has been renamed to `BACKEND` in order to clarify that it is set to select a backend to connect to.

This PR also removes the Webpack EnvironmentPlugin, which was previously used to make available environment variables in client-side Javascript. Instead, `webpack-dotenv` is used.

Closes https://github.com/FixMyBerlin/fixmy.platform/issues/285

### CODE REVIEW

**Comment by ciex** [comment]
@geomC hier habe ich mal dotenv konfiguriert. Der build failt, weil ich das command aus der package.json rausgenommen habe.

**Comment by geomC** [comment]
> @geomC hier habe ich mal dotenv konfiguriert.

Sehr schön :) Ich hoffe, ich finde am WE die Zeit um kurz drüber zu schauen.

**Comment by geomC** [comment]
Sieht erstmal gut aus! Nettes refactoring, vor allem die deutlich schlankere [package.json](https://github.com/FixMyBerlin/fixmy.frontend/pull/332/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2). Falls ich wie gesagt die Zeit finde, check ichs mal aus und teste durch.

File: cypress/support/commands.js [hunk 1]
```diff
@@ -75,9 +75,14 @@ Cypress.Commands.add('fmbGoToScene', (scene = 1) => {
  */
 Cypress.Commands.add('fmbLogin', () => {
   const credentials = {
-    username: Cypress.env('username'),
-    password: Cypress.env('password')
+    username: Cypress.env('CYPRESS_USERNAME'),
+    password: Cypress.env('CYPRESS_PASSWORD')
   };
+  if (credentials.username == null || credentials.password == null) {
```
Comment (geomC) [inline cypress/support/commands.js]: good stuff

**Pull request merged**